src/test/kotlin/net/atos/zac/admin/ZaakafhandelParameterBeheerServiceTest.kt

```diff
@@ -172,4 +177,30 @@ class ZaakafhandelParameterBeheerServiceTest : BehaviorSpec({
             }
         }
     }
+
+    Given("A Zaak is aangepast") {
+        val zaaktypeUri = URI("test-url");
+
+        every { ztcClientService.readZaaktype(zaaktypeUri) } returns ZaakType()
+
+        Then("The HumanTaskParameters should get copied") {
+
```

Comment:
empty test 😞

Reply:
yes, it was my attempt to start the testing but then I ran out of time. so tbc